playground/components/editor.tsx

```diff
@@ -14,10 +13,10 @@ import { MonacoEditor } from './monaco-editor';
 import { GridResizer } from './gridResize';
 import { useRef, useState } from 'react';
 import cls from 'classnames';
+import { FrameworkSwitcher } from './framework-switcher';
+import { useAtomValue, atom } from 'jotai';
 
-type Props = {
-  framework?: Frameworks;
-};
+export const frameworkAtom = atom<Frameworks>('react');
```

Comment:
IMHO, it is better to use context + useState pattern over `jotai`. We are only trying to prevent the prop drilling and `jotai` is overkill.

Reply:
We had a parallel discussion in Discord, my bad for not surfacing it in the PR. The Jotai move is mostly as because we expect we will go quickly have more state come up.

100% agree that this can be simply context + useState, I personally don't have a preference over each but I think Jotai will come to make sense in the next 2-3 features.